x-pack/plugins/monitoring/server/lib/alerts/fetch_index_shard_size.ts

```diff
@@ -26,7 +26,7 @@ const memoizedIndexPatterns = (globPatterns: string) => {
   ) as RegExPatterns;
 };
 
-const gbMultiplier = 1000000000;
+const gbMultiplier = Math.pow(1024, 3);
```

Comment:
Could we add a test case to validate the what's expected after this change?

Reply:
added test _higher alert threshold_